bitnami/rabbitmq/Chart.yaml

```
@@ -30,4 +30,4 @@ maintainers:
 name: rabbitmq
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/rabbitmq
-version: 12.4.2
+version: 12.4.3
```

I think is better to use a minor update because we are adding a new feature.
```suggestion
version: 12.5.0
```